config/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "monitoringCluster": "mad",
+  "logDirectory": "logs",
+  "pipelinesDirectory": "/opt/mad/config/pipelines",
+  "httpHost": "0.0.0.0",
+  "httpPort": 7090,
+  "httpHealthCheckPath": "/ping",
+  "httpStatusPath": "/status.json",
+  "jvmMetricsCollectionInterval": "PT.5S",
+  "akkaConfiguration": {
+    "akka": {
+      "loggers": [
+        "akka.event.slf4j.Slf4jLogger"
+      ],
+      "loglevel": "DEBUG",
```

Comment:
Yeah, it just gets uncached fro time to time, me we touch it so infrequently. It makes you  want to use hocon for config (w. comments).